packages/xstate-test/src/types.ts

```diff
@@ -1,5 +1,5 @@
 import { EventObject, State, StateNode } from 'xstate';
-export interface TestMeta<T, TContext> {
+export interface TesTTypesMeta<T, TContext> {
```

Comment:
Is this a typo?

Reply:
Ye, somehow this got south despite me using `Rename Symbol` functionality